staging/src/k8s.io/apiserver/pkg/apis/example/types.go

```diff
@@ -33,7 +33,7 @@ type (
 type Pod struct {
 	metav1.TypeMeta
 	// +optional
-	metav1.ObjectMeta
+	metav1.ObjectMeta `json:"metadata,omitempty"`
```

Comment:
Why do you add this one?

I think we should be consistent - either add json tags to all fields, or to any of those.

Reply:
For test https://github.com/dixudx/kubernetes/blob/b7763c4cce9b4190093fdb346debe53fff39f79f/staging/src/k8s.io/apiserver/pkg/storage/etcd/etcd_watcher_test.go#L132.